src/NodeRTLib/ProjectTemplates/binding.gyp

```diff
@@ -51,7 +51,9 @@
           "VCCLCompilerTool": {
             "AdditionalUsingDirectories": [
               "%ProgramFiles(x86)%/Microsoft Visual Studio 14.0/VC/lib/store/references",
-              "%ProgramFiles%/Microsoft Visual Studio 14.0/VC/lib/store/references"
+              "%ProgramFiles%/Microsoft Visual Studio 14.0/VC/lib/store/references",
+              "%ProgramFiles(x86)%/Microsoft Visual Studio/2019/Community/VC/Tools/MSVC/14.24.28314/lib/x86/store/references",
```

Comment:
Isn't this path specific to VS Community version ? Enterprise and Professional version will have different path.